update/ansible/playbook/tasks/roles/nginx/files/conf.d/pmm.conf

```diff
@@ -43,6 +43,29 @@
     ssl_trusted_certificate /srv/nginx/ca-certs.pem;
     ssl_dhparam /srv/nginx/dhparam.pem;
 
+    # this block checks for maintenance.html file and, if it exists, it redirects all requests to the maintenance page
+    # there are two exceptions for it /v1/Updates/Status and /auth_request endpoints
+    set $skip_maintenance 0;
```

Comment:
I think the variable is somewhat of a misnomer, it seems like it should be `maintenance_mode` or `maintenance_on`. WDYT?